tools/myst/host/exec.c

```diff
@@ -213,6 +213,34 @@ static void _wait_on_child_threads(void)
     }
 }
 
+#define EXTENDED_FEATURE_FLAGS_FUNCTION 0x7
+void _print_platform_details()
+{
+    uint32_t rax = 0;
+    uint32_t rbx = 0;
+    uint32_t rcx = 0;
+    uint32_t rdx = 0;
+
+    myst_cpuid_ocall(
```

Comment:
Why do we need a "ocall"?

Reply:
Thats the existing cpuid query function.